src/core/server/elasticsearch/elasticsearch_config.ts

```diff
@@ -412,6 +413,28 @@ export class ElasticsearchConfig {
       verificationMode,
     };
   }
+
+  public toJSON() {
```

Comment:
We can't serialize functions so we need to turn the config into a pojo before we can share it with the worker. I don't think there's any benefit to having an actual class contain the Elasticsearch config other than convention. So I'd lean towards just turning this class into a function that takes config and returns a type plain object.

We also can't serialize moment values, so these need to be converted. Our code already deals with both moment/concrete values but I think we might as well turn everything into ms as soon as we read the config, I don't think we really ever convert these duration values into other formats.